api/controllers/orderController.js

```diff
@@ -0,0 +1,27 @@
+const orderService = require("../services/orderService");
+const { asyncErrorHandler } = require("../middleware/errorHandling");
+
+const order = asyncErrorHandler(async (request, response) => {
+  const userId = request.userId;
+  const { cartId } = request.query;
+  const { totalPrice, paymentMethod } = request.body;
+  await orderService.order(userId, cartId, totalPrice, paymentMethod);
+  return response.status(200).send("orderSuccess");
+});
+
+const loadOrderStatus = asyncErrorHandler(async (req, res) => {
```

Comment:
- route 쪽에도 남겼지만, 주문 내역을 가져오는 것이 필요하지 않을까요?
- load 보다는 get이 조금 더 데이터를 가져오는 함수 이름으로 적절할 것 같아요.
```suggestion
const getOrderStatus = asyncErrorHandler(async (req, res) => {
```